collapsiblecalendarview2/src/main/java/com/hyperexternal/collapsiblecalendarview/widget/CollapsibleCalendar.kt

```diff
@@ -330,16 +329,6 @@ open class CollapsibleCalendar : UICalendar, View.OnClickListener {
                         imgEventTag.visibility = View.INVISIBLE
                     } else {
                         view.setOnClickListener { v -> onItemClicked(v, mAdapter.getItem(i)) }
-                        val parent = view.getParent() as View?
-                        parent?.post {
-                            val r = Rect()
-                            view.getHitRect(r)
-                            r.top -= context.dipToPixels(20).toInt()
-                            r.bottom += context.dipToPixels(20).toInt()
-                            r.left -= context.dipToPixels(20).toInt()
-                            r.right += context.dipToPixels(20).toInt()
-                            parent.touchDelegate = TouchDelegate(r, view)
-                        }
```

Comment:
Same as above.

Reply:
this code was written by kgoel in an attempt to increase the touch targets of the days of the calendar but it doesnt work so I am removing it. It doesnt work because each day the touch target is already as big as it can be and all the days of the month are next to each other so increasing touch target size just makes them all overlap each other accomplishing nothing